### PR TITLE
Audit Logs: Revamp of audit logs application

### PR DESCRIPTION
**Pull request by adamzinder**

### CODE REVIEW

File: ruby-audit-logs-example/views/send_events.erb [hunk 1]
```diff
@@ -1,102 +1,220 @@
...
+                        <div class="flex space-evenly width-11vw content-button tab">
+                            <img src="<%= url("images/share-2.svg")%>" class="icon" alt="settings icon">
                             <div>
-                                <button class="box" name="event" id="user_org_deleted" value="2" type="submit" onclick="showSnackbar()">                            
-                                    <div class="flex width-100p">
-                                        <p>User Organization Deleted</p> 
-                                    </div>                           
-                                </button>                    
-                                <button class="box" name="event" id="user_connection_deleted" value="3" type="submit" onclick="showSnackbar()">                            
+                                <a href="/events?intent=log_streams&organization_id=<%=@organization.id%>" class="remove-style"><button class="remove-style">Configure Log Streams</button></a>
```
Comment (awolfden) [inline ruby-audit-logs-example/views/send_events.erb:83]: Getting this error when clicking the Log Streams button. Can you update the intent to be "audit_logs"?
CleanShot 2023-03-27 at 14 09 50

File: ruby-audit-logs-example/views/send_events.erb [hunk 2]
```diff
@@ -1,102 +1,220 @@
...
-                </div>
-                <div class="card width-40vw">
-                    <div class="flex space-evenly width-40vw margin-b-25">
-                        <div>
-                            <div class="button button-outline width-15vw">
-                                <a href="https://workos.com/docs/audit-logs" target="_blank"><p>Audit Logs Docs</p></a>                            
-                            </div>                    
+                </div>                
+            </div>
+            <div class="card events-card width-65vw height-45vh">
```
Comment (awolfden) [inline ruby-audit-logs-example/views/send_events.erb:59]: Can you remove the height-45vh class here? It's causing the card to chop off the content. 
CleanShot 2023-03-27 at 14 14 17

**Comment by awolfden** [comment]
Could you update the readme to go over the new app flow? ex: only one audit log event needs to be added and no org ID gets manually entered.

File: ruby-audit-logs-example/app.rb [hunk 3]
```diff
@@ -22,77 +23,118 @@
...
-        $user_connection_deleted,
-    ]
-
-  event = events[event_type.to_i]
+get '/events' do
+  link = WorkOS::Portal.generate_link(
+    organization: params[:organization_id],
+    intent: params[:intent],
+    )
+  redirect link
```
Comment (awolfden) [inline ruby-audit-logs-example/app.rb:59]: After sending a successful event I got redirected back to the home page where you pick an organization. Can we redirect back to the organization's page you were just on after you send an event?

Comment (awolfden) [reply ruby-audit-logs-example/app.rb:59]: CleanShot 2023-03-27 at 14 22 29

**Review by awolfden — CHANGES_REQUESTED** [review]
Overall looks great! Just a few comments/revisions.

File: ruby-audit-logs-example/app.rb (hunk 3 above)
Comment (adamzinder) [reply ruby-audit-logs-example/app.rb:59]: done

**Review by awolfden — APPROVED** [review]
superb 👍🏽

**Pull request merged**